watertap/examples/flowsheets/full_treatment_train/electrolyte_scaling_utils.py

```diff
@@ -190,31 +190,5 @@ def calculate_chemical_scaling_factors(unit, thermo_params, rxn_params, state_ar
     calculate_chemical_scaling_factors_for_energy_balances(unit)
     calculate_chemical_scaling_factors_for_material_balances(unit)
 
-    # If calling multiple times, then this causes errors
-    #   Catch those errors here and move on (still need to
-    #       have the setting of suffixes and the auto_scale_jac function)
-    try:
-        iscale.calculate_scaling_factors(unit)
-    except:
-        pass
-
     flags = fix_state_vars(unit.control_volume.properties_out, state_args)
     revert_state_vars(unit.control_volume.properties_out, flags)
-
-    iscale.constraint_autoscale_large_jac(unit)
-
-    if output_jac == True:
-        jac, nlp = iscale.get_jacobian(unit, scaled=True)
-        print("Extreme Jacobian entries:")
-        for i in iscale.extreme_jacobian_entries(jac=jac, nlp=nlp, large=100):
-            print(f"    {i[0]:.2e}, [{i[1]}, {i[2]}]")
-        print("Unscaled constraints:")
-        for c in iscale.unscaled_constraints_generator(unit):
-            print(f"    {c}")
-        print("Scaled constraints by factor:")
-        for c, s in iscale.constraints_with_scale_factor_generator(unit):
-            print(f"    {c}, {s}")
-        print("Badly scaled variables:")
-        for v, sv in iscale.badly_scaled_var_generator(unit, large=1e2, small=1e-2, zero=1e-12):
-            print(f"    {v} -- {sv} -- {iscale.get_scaling_factor(v)}")
-        print(f"Jacobian Condition Number: {iscale.jacobian_cond(jac=jac):.2e}")
```

Comment:
Much of this is removed already in the pending PR #284 (there will be many conflicts after 284 is merged)